baselines/dom.generated.d.ts

```diff
@@ -3073,7 +3073,7 @@ interface Blob {
 
 declare var Blob: {
     prototype: Blob;
-    new(blobParts?: BlobPart[], options?: BlobPropertyBag): Blob;
+    new(blobParts?: Iterable<BlobPart>, options?: BlobPropertyBag): Blob;
```

Comment:
You cannot do this as the base lib needs to be ES5 compatible. This kind of thing is done in dom.iterable but the constructor can't be extended in the current design. We are blocked here.